src/main/kotlin/no/nav/familie/tilbake/e2e/AutotestController.kt

```diff
@@ -0,0 +1,38 @@
+package no.nav.familie.tilbake.e2e
+
+import no.nav.familie.prosessering.domene.Task
+import no.nav.familie.prosessering.domene.TaskRepository
+import no.nav.familie.tilbake.kravgrunnlag.task.BehandleKravgrunnlagTask
+import no.nav.familie.tilbake.kravgrunnlag.task.BehandleStatusmeldingTask
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.context.annotation.Profile
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.util.Properties
+import java.util.UUID
+
+@RestController
+@RequestMapping("/api/autotest")
+@ProtectedWithClaims(issuer = "azuread")
+@Profile("e2e","local","integrasjonstest")
+class AutotestController (private val taskRepository: TaskRepository) {
```

Comment:
Er det mulig å opprette disse tjenestene direkte i familie-tilbake-e2e repo?

Reply:
Er mulig, å gå direkte mot databasen til familie-tilbake, men da blir det vanskeligere å holde tasktypene i synk, i tillegg må e2e kjenne til intern struktur for familie-tilbake og det synes jeg ikke er helt heldig.